musicbot/bot.py

```diff
@@ -114,7 +114,7 @@ def __init__(self, config_file=None, perms_file=None, aliases_file=None):
             "auto_paused": False,
             "availability_paused": False,
             "guild_id": None,
```

Comment:
We can probably remove this now.

Reply:
Not removing `availability_paused` because it is still used in line 4668, 4674 and 4686